pkg/chaosctl/common/common.go

```diff
@@ -153,11 +151,15 @@ func MarshalChaos(s interface{}) (string, error) {
 
 // InitClientSet inits two different clients that would be used
 func InitClientSet() (*ClientSet, error) {
-	ctrlClient, err := client.New(config.GetConfigOrDie(), client.Options{Scheme: scheme})
+	restconfig, err := loadRestConfig()
```

Comment:
Why not using `config.GetConfigOrDie()` directly?

Reply:
When this PR just created, the refine about logging are not merged into master. The `logger` of controller-runtime is not configured. So `config.GetConfigOrDie()` will return without any output.

And `os.Exit(1)` makes `defer func(){}()` will not execute anymore.  I think it's not so good.🤪 I prefer to return an error then handle it at the upper layer, instead of exit directly.

I just realize that we could `config.GetConfig()`. It meets all my requirements.